source/Weapon.cpp

```diff
@@ -474,3 +503,17 @@ double Weapon::TotalDamage(int index) const
 	}
 	return damage[index];
 }
+
+
+
+std::pair<Distribution::Type, bool> Weapon::InaccuracyDistribution() const
+{
+	return std::make_pair(inaccuracyType, inaccuracyInversion);
```

Comment:
```suggestion
	return make_pair(inaccuracyType, inaccuracyInversion);
```

Reply:
Good catch.  The return no longer uses make_pair as of e896b21fc1964d188d3bf6cea36bfde2a4b2d756, but I'll modify the function declaration similarly.